In [1]:
import pandas as pd
import numpy as np

In [16]:
file_path = '..\BraTS20 Training Metadata.csv'
images_path = 'content\data'

In [11]:
df = pd.read_csv(file_path)
df.shape

(57195, 8)

In [12]:
df.head()

slice_path  target  volume  slice  \
0  ../input/brats2020-training-data/BraTS2020_tra...       0      41      0   
1  ../input/brats2020-training-data/BraTS2020_tra...       0      41      1   
2  ../input/brats2020-training-data/BraTS2020_tra...       0      41      2   
3  ../input/brats2020-training-data/BraTS2020_tra...       0      41      3   
4  ../input/brats2020-training-data/BraTS2020_tra...       0      41      4   

   label0_pxl_cnt  label1_pxl_cnt  label2_pxl_cnt  background_ratio  
0               0               0               0               1.0  
1               0               0               0               1.0  
2               0               0               0               1.0  
3               0               0               0               1.0  
4               0               0               0               1.0

In [13]:
import os
import shutil

In [18]:
target_dir = 'content/targetdata'

if not os.path.exists(target_dir):
    os.mkdir(target_dir)
    print('Target folder as created')
    



Target folder as created


In [19]:
cnt = 0
for (i,row) in df.iterrows():
    if row['target'] == 1:
#         filename = row['slice_name']
#         image_path = os.path.join(images_path)
        cnt +=1
print(cnt)

24422


# total target we have value 1 = 24422
# target have value 0 = 32773

In [22]:
iris = load_iris()

NameError: name 'load_iris' is not defined